jobserver/middleware.py

```diff
@@ -39,7 +38,9 @@ def process_template_response(self, request, response):
 
         # flatten template_name to a string
         if isinstance(template_name, list):
-            template_name = first(template_name)
+            template_name = next(  # pragma: no branch
+                (t for t in template_name if t), None
+            )
```

Comment:
thought: This is the existing behaviour.

Should this fail more loudly if there is no template name found? Is that ever possible?